resident/resident-service/src/main/java/io/mosip/resident/service/impl/IdentityServiceImpl.java

```diff
@@ -570,8 +571,10 @@ public String getIndividualIdType(String individualId){
 			return UIN;
 		} else if(requestValidator.validateVid(individualId)){
 			return VID;
-		} else{
+		} else if(requestValidator.validateRid(individualId)){
 			return AID;
+		} else {
+			throw new InvalidIDException(ResidentErrorCode.INVALID_ID.getErrorCode(), ResidentErrorCode.INVALID_ID.getErrorMessage());
```

Comment:
Correct the error: Invalid input parameter - individualId

Reply:
done.